internal/backend/azure/config.go

```diff
@@ -51,5 +57,11 @@ func ParseConfig(s string) (interface{}, error) {
 	cfg := NewConfig()
 	cfg.Container = container
 	cfg.Prefix = prefix
+
+	cfg.ClientID = os.Getenv("AZURE_CLIENT_ID")
```

Comment:
Also please document the variables at https://github.com/restic/restic/blob/master/doc/040_backup.rst#environment-variables